src/Particular.ThroughputCollector.Persistence/IThroughputDataStore.cs

```diff
@@ -15,11 +15,14 @@ public interface IThroughputDataStore
 
     Task SaveEndpoint(Endpoint endpoint, CancellationToken cancellationToken = default);
 
-    Task RecordEndpointThroughput(EndpointIdentifier id, IEnumerable<EndpointDailyThroughput> throughput, CancellationToken cancellationToken = default);
+    Task<IDictionary<string, IEnumerable<ThroughputData>>> GetEndpointThroughputByQueueName(IEnumerable<string> queueNames, CancellationToken cancellationToken = default);
```

Comment:
I am not sold on the return type.
To me, it would make more sense to be something like `IAsyncEnumerable<(string queuename, IAsyncEnumerable<ThroughputData> throughput)>`

Reply:
https://github.com/orgs/Particular/projects/197/views/1?pane=issue&itemId=58620925